openapi.yaml

```diff
@@ -2165,6 +2177,634 @@ paths:
+          content: {}
+        409:
+          description: Conflict with current state
+      security:
+        - oscal_auth:
+            - write:catalogs
+            - read:catalogs
+      x-codegen-request-body-name: body
+    put:
+      tags:
+        - OSCAL Plan of Action and Milestones
+      summary: Replaces an existing OSCAL Plan of Action and Milestones
+      operationId: replacePoam
+      parameters:
+        - name: catalogId
+          in: path
+          description: ID of catalog to replace.
+          required: true
+          schema:
+            type: string
+      requestBody:
+        description: Catalog object to be replaced.
+        content:
+          application/json:
+            schema:
+              $ref: "#/components/schemas/OSCALCatalog"
+        required: true
+      responses:
+        200:
+          description: Updated catalog
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/OSCALCatalog"
+        400:
+          description: Bad Request
+        404:
+          description: Catalog not found
+        409:
+          description: Conflict with current state
+        415:
+          description: Unsupported media type
+      security:
+        - oscal_auth:
+            - write:catalogs
+            - read:catalogs
+      x-codegen-request-body-name: body
+    delete:
+      tags:
+        - OSCAL Plan of Action and Milestones
+      summary: Deletes an OSCAL Plan of Action and Milestones
+      operationId: deletePoam
+      parameters:
+        - name: api_key
+          in: header
+          schema:
+            type: string
+        - name: catalogId
+          in: path
+          description: Catalog ID to delete
+          required: true
+          schema:
+            type: string
+      responses:
+        400:
+          description: Invalid ID supplied
+          content: {}
+        404:
+          description: Catalog not found
+          content: {}
+      security:
+        - oscal_auth:
+            - write:catalogs
+            - read:catalogs
+  /plan-of-action-and-milestones/search:
+    get:
+      tags:
+        - OSCAL Plan of Action and Milestones
+      summary: Searches for OSCAL Plan of Action and Milestones
+      operationId: findPoamsByName
+      parameters:
+        - name: query
+          in: query
+          description: Terms to search for in catalog names
+          required: true
+          schema:
+            type: string
+      responses:
+        200:
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/OSCALCatalog"
+        400:
+          description: Invalid status value
+          content: {}
+      security:
+        - oscal_auth:
+            - read:catalogs
+
+  /system-assessment-plans:
```

Comment:
Swagger Editor is showing a structural error here, and I'm not sure why. This is also happening on line 2598.

Reply:
"Structural error at paths./system-assessment-plans should NOT have additional properties additionalProperty: /system-assesment-plans/{catalogId}, /system-assesment-plans/search"